api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -133,20 +138,108 @@ private void handleHome(RoutingContext ctx) {
     response.end("<h1>go to /login with query string</h1>");
   }
 
+  private void handleLogin(RoutingContext ctx) {
+    HttpServerResponse response = ctx.response();
+    Session session = ctx.session();
```

Comment:
I think we're still using sessions for some data, so we will need sessions just not for authorization